test-dp-submodule/stick-breaking-weights-rv.py

```diff
@@ -0,0 +1,73 @@
+import numpy as np
+import pymc3 as pm # v4.0
+
+from aesara import tensor as at # v2.0.12
+
+import aesara, warnings
+
+from aesara.tensor.random.op import RandomVariable
+
+from pymc3.distributions.continuous import assert_negative_support
+from pymc3.distributions.distribution import Continuous
+
+
+class StickBreakingWeightsRV(RandomVariable):
+    name = "stick_breaking_weights"
+    ndim_supp = 1
```

Comment:
```suggestion
    ndim_supp = 0
```
Can you try this and see if this works?

Reply:
I think that this is correct... I tried most combinations of `ndim_supp = 0, ndims_params = [1]`, etc., but ultimately I believe that `alpha` needs to be a vector. I am getting a new error, but at least I am able to get to the NUTS initialization step 😅

Reply:
Thanks so much!!